spec/features/movies/show_spec.rb

```diff
@@ -2,17 +2,30 @@
 
 RSpec.describe "Movie Show Page" do
   describe "As an authenticated user" do
-    before :each do
+    it "I can see movie details", :vcr do
       @user = User.create!(user_id: "100000000000000000000", name: "John Smith")
       allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
-      visit movies_path
-      click_on "Top Rated"
-      click_on "Gabriel's Inferno Part II"
-      expect(current_path).to eq('/movies/724089')
-    end
 
-    it "I can see movie details" do
-      expect(page).to have_content("")
+      cassette = 'spec/fixtures/vcr_cassettes/Movie_Show_Page/As_an_authenticated_user/I_can_see_movie_details.yml'
+      movie_details = File.read(cassette)
+      yaml = YAML.load(movie_details, symbolize_names: true)
+      json = yaml[:http_interactions][0][:response][:body][:string]
+      movie_params = JSON.parse(json, symbolize_names: true)
+      movie = Movie.new(movie_params)
+
+      visit("/movies/#{movie.tmdb_id}")
+
+      expect(page).to have_content(movie.title)
+      expect(page).to have_content(movie.vote_average)
+      expect(page).to have_content(movie.runtime)
+      expect(page).to have_content(movie.genres.join(', '))
+      expect(page).to have_content(movie.cast_members.first.actor)
+      expect(page).to have_content(movie.cast_members.first.character)
+      expect(page).to have_content(movie.description)
+      expect(movie.reviews_count).to have_content(0)
+      expect(page).to have_content(movie.reviews_count)
+      # expect(page).to have_content(movie.reviews.first.author)
```

Comment:
Do we have to use a different mock since this one doesn't have any movie reviews?

Reply:
Maybe... 👀